contract/__manifest__.py

```diff
@@ -17,7 +17,13 @@
               "LasLabs, "
               "Odoo Community Association (OCA)",
     'website': 'https://github.com/oca/contract',
-    'depends': ['base', 'account', 'product', 'portal'],
+    'depends': [
+        'base',
+        'account',
+        'product',
+        'portal',
+        'email_template_qweb'
```

Comment:
This dependency is too hard and can't be added now. When I told you about QWeb, I meant to use core feature of enveloping a QWeb

Reply:
OK, change now to `message_post_with_view` function